### ISSUE TITLE
RemoteControl API returns invalid message type 80/71 (HTTP/0.9?)

### ISSUE THREAD

**Issue by FogoVoar** [Stellarium/stellarium#3762]
### Expected Behaviour
Manipulating camera orientation in Stellarium by sending a POST request to the RemoteControl plugin server.

### Actual Behaviour
With curl it returns an invalid HTTP/0.9 response not supported by default, 
When specifying --http0.9 it returns the "invalid message type 80" error and sometimes "invalid message type 71" amid some invalid characters.

### Steps to reproduce
1. Enable the [RemoteControl API](https://stellarium.org/doc/head/remoteControlApi.html) plugin.
2. Adjust the port if necessary (20180 in this example).
3. Start the server and run the curl command: `curl --data 'j2000=[0.5,0.3,0.2]' http://192.168.xx.xx:20180/api/main/view` (where x is your local IP)

![image](https://github.com/Stellarium/stellarium/assets/112202663/d94a0edc-3580-4907-97ce-7bd449d9cb5f)

By appending `--http0.9` like:
`curl --http0.9 --data 'j2000=[0.5,0.3,0.2]' http://192.168.xx.xx:20180/api/main/view --output output.txt`
output.txt returns:
```
� 5StellariumSyncPluginProtocol�   � �� È�°ª
#Gm˜êå»•p‡l  �   �invalid message type 80
```

### System
* Stellarium version: stellarium-24.1-qt6-win64
* Operating system: Ubuntu-24.04 running on WSL2 (Windows 10 Pro, 64, v22H2)
* CURL version: `curl 8.5.0 (x86_64-pc-linux-gnu) libcurl/8.5.0 OpenSSL/3.0.13 zlib/1.3 brotli/1.1.0 zstd/1.5.5 libidn2/2.3.7 libpsl/0.21.2 (+libidn2/2.3.7) libssh/0.10.6/openssl/zlib nghttp2/1.59.0 librtmp/2.3 OpenLDAP/2.6.7`.
Also tested with Ubuntu-22.04.3 and curl 7.81.0, no luck.
(no luck with nodejs axios either)

### Logfile 
(I omitted some unrelated lines from the log file):
[log.txt](https://github.com/user-attachments/files/15571005/log.txt)

This issue comes from [this discussion](https://github.com/Stellarium/stellarium/discussions/3760).

**Comment by gzotti** [comment]
Still cannot reproduce this with Stellarium 24.1+ (current master), Ubuntu 22.04.4 on W11home 23H2/WSL2.1.5.0, curl 7.81.0. Works as always. Ports 8090 and 20180 and some others.

**Comment by gzotti** [comment]
What happens with --http1.0 ?

**Comment by FogoVoar** [comment]
> What happens with --http1.0 ?

Returns `curl: (1) Received HTTP/0.9 when not allowed` too...

**Comment by gzotti** [comment]
Is there an alternative program? I mean, in the end you wanted to contact Stellarium from your program. Curl was meant for a quick test.

**Comment by FogoVoar** [comment]
> Is there an alternative program? I mean, in the end you wanted to contact Stellarium from your program. Curl was meant for a quick test.

For the application I had in mind I would probably use nodejs with fetch or axios, this is the fetch POST output:
![image](https://github.com/Stellarium/stellarium/assets/112202663/89acf7fd-f451-4134-895a-a9c80c25ffa1)

**Comment by gzotti** [comment]
Weird. Stellarium's RemoteControl provides a web interface, is connected from APT, my Unity bridge, and a few other programs. I have no idea what's going on in your system.

**Comment by FogoVoar** [comment]
Ok, I found the "problem", and it's my fault.
I was mistakenly configuring "Remote Sync" instead of "Remote Control", since I'm using the PT-BR interface, I assumed that "Sincronização Remota" was the translation they had given for "Remote Control", obviously it isn't, a little above the list there is the "Controle Remoto" ("Remote Control") plugin which is the right plugin.
![image](https://github.com/Stellarium/stellarium/assets/112202663/33c0cb74-c56c-4fc4-8d00-65d41abdcb6b)

In the English version it is easy to get the right one because both plugins are very close.
![image](https://github.com/Stellarium/stellarium/assets/112202663/ac2d76e0-7a81-43ea-9ed8-96ab76d75473)

Well, I can only apologize for the time we wasted trying to debug a problem that doesn't exist, sorry! 🤦‍♂️
